### PR TITLE
Left UI set up (issue with width of bg color)

### PR DESCRIPTION
**Pull request by Mclblack**

### CODE REVIEW

File: styles/LeftStyles.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import {StyleSheet} from 'react-native';
+
+var styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        justifyContent: "center",
+        alignItems: "center",
+        backgroundColor:"pink"
+
```
Comment (eliastayag) [inline styles/LeftStyles.js]: Try adding width and making it 100%

**Comment by j-ulienguyen** [comment]
try giving main comp width of 100% as well

**Review by j-ulienguyen — APPROVED** [review]
Looks good now

**Comment by j-ulienguyen** [comment]
Ill fix the conflicts and push it up

**Pull request merged**